### DISCUSSION TITLE
classes/class_codehighlighter

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#202] [User-contributed notes]
# classes/class_codehighlighter

Inherits: SyntaxHighlighter< Resource< RefCounted< Object A syntax highlighter intended for code. Description: By adjusting various properties of this resource, you can change the colors of strings...

https://docs.godotengine.org/en/stable/classes/class_codehighlighter.html

**Comment by awildergoose** [comment]
Please someone add examples to the documentation of this class.

**Reply by Calinou** [reply]
Please open an issue on godot-docs for documentation requests: https://github.com/godotengine/godot-docs/issues
